package.json

```diff
@@ -103,6 +104,7 @@
     "ava": "0.25.0",
     "map-stream": "0.0.7",
     "pre-commit": "1.2.2",
-    "vinyl-string": "1.0.2"
+    "vinyl-string": "1.0.2",
+    "lodash": "latest"
```

Comment:
Why did you choose latest instead of a version?

Reply:
argh - sorry. That was automatically added by my IDE, because it was missing in the package.json but used in config.js line 188. You can change this as needed.